package.json

```diff
@@ -4,6 +4,7 @@
   "description": "CLI bar charts",
   "main": "lib/babar.js",
   "scripts": {
+    "develop": "./node_modules/coffee-script/bin/coffee --watch -o lib/ -c src/babar.coffee",
```

Comment:
The leading `./node_modules/coffee-script/bin/` is redundant here. Locally installed modules are automatically on `$PATH` for npm scripts, and should be used directly as install locations are not reliable.